broker.properties

```diff
@@ -22,4 +22,6 @@ samplebroker.core.jmsManagementService.jmsBrokerUrl = tcp://localhost:61616
 samplebroker.core.powerTacBroker.retryTimeLimit = 400000
 # by default, this module is interactive, which means the peer
 # must send the pause-release message at the end of every timeslot.
-#samplebroker.core.powerTacBroker.interactive = true
\ No newline at end of file
+#samplebroker.core.powerTacBroker.interactive = truesamplebroker.core.brokerMessageReceiver.rawXml = true
+samplebroker.core.brokerMessageReceiver.rawXml = true
+samplebroker.core.brokerMessageReceiver.xmlForwardType = SocketAdapter
```

Comment:
Configuration via broker.properties, linked to [this PR](https://github.com/powertac/powertac-core/pulls)

Reply:
Line 25 of broker.properties seems like you lost a line ending.